docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -84,6 +89,40 @@ eland_import_hub_model
 
 For more details, refer to https://github.com/elastic/eland#nlp-with-pytorch.
 
+[discrete]
+[[ml-nlp-import-docker]]
+=== Import with Docker
+
+IMPORTANT: To use the Docker container, you need to clone the Eland repository: 
+https://github.com/elastic/eland
+
+If you want to use Eland without installing it, build the Docker container to 
+run the available scripts:
```

Comment:
Its not obvious that the following command is run from the root of the cloned Eland repo.

```suggestion
If you want to use Eland without installing it, clone the Eland repository and from the root directory run the following to build the Docker container:
```